src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowPredicate.java

```diff
@@ -491,15 +496,156 @@ private static boolean getPredicateForResponseTypeSpec(CaseData caseData, Respon
                 predicate = basePredicate;
                 break;
             case TWO_V_ONE:
-                predicate = responseType.equals(caseData.getClaimant1ClaimResponseTypeForSpec())
-                    && responseType.equals(caseData.getClaimant2ClaimResponseTypeForSpec());
+                if (YES.equals(caseData.getDefendantSingleResponseToBothClaimants())) {
+                    predicate = responseType.equals(caseData.getRespondent1ClaimResponseTypeForSpec());
+                } else {
+                    predicate = responseType.equals(caseData.getClaimant1ClaimResponseTypeForSpec())
+                        && responseType.equals(caseData.getClaimant2ClaimResponseTypeForSpec());
+                }
                 break;
             default:
                 break;
         }
         return predicate;
     }
 
+    public static final Predicate<CaseData> divergentRespondWithDQAndGoOfflineSpec = caseData ->
+        isDivergentResponsesWithDQAndGoOfflineSpec(caseData);
+
+    private static boolean isDivergentResponsesWithDQAndGoOfflineSpec(CaseData caseData) {
+
+        if (!SPEC_CLAIM.equals(caseData.getSuperClaimType())) {
+            return false;
+        }
+
+        switch (getMultiPartyScenario(caseData)) {
```

Comment:
The 3 cases require that respondent1ClaimResponseTypeForSpec and respondent2ClaimResponseTypeForSpec are different. You can check that before the switch and simplify a bit the three blocks.

Reply:
Left to a less time-pressed occasion